plugins/headlamp-plugin/config/webpack.config.js

```diff
@@ -10,6 +10,7 @@ const externalModules = {
   notistack: 'pluginLib.Notistack',
   '@kinvolk/headlamp-plugin/lib/CommonComponents': 'pluginLib.CommonComponents',
   '@kinvolk/headlamp-plugin/lib': 'pluginLib',
+  notification: 'pluginLib.Notification',
```

Comment:
This is not needed here. You already added the Notification to the pluginLib above.

Reply:
I don't get this, we have other modules here which are also added to the pluginLib how does it work here??